### ISSUE TITLE
keen slider disabling double tap/pinch to zoom in/out?

### ISSUE THREAD

**Issue by rizabek-mukanov** [rcbyr/keen-slider#237]
i have modal with images where every image is keen slider slide. apparently double tap and pinch are not working to zoom. 
is it intentionally done by lib? i was thinking keen slider itself handle slide events and thats why it disables some other events

**Comment by rcbyr** [comment]
Hey @rizabek-mukanov 

you just have to remove the `keen-slider` class and add the minimum needed styles to the wrapper. See here: https://codesandbox.io/s/angry-firefly-s2cm8g?file=/src/App.js:243-290

**Comment by pySilver** [comment]
@rcbyr thanks for your work. What is the reason behind tweaking touch-actions by default? Is this a mere usability thing?

**Comment by kj** [comment]
@rcbyr I tried your example in both Firefox for Android and Brave for Android and I couldn't use pinch zoom in either. This is the one thing that's missing for me in this library. The only slider libraries I've found pinch zoom to work with are [Swiper](https://swiperjs.com/demos/300-thumbs-gallery/core) and [Glider.js](https://nickpiscitelli.github.io/Glider.js/). I'd love to see this work in Keen-Slider too.
